CharlieBackend.Api/Validators/AccountDTOValidators/AuthenticationDtoValidator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using CharlieBackend.Core.DTO.Account;
+using FluentValidation;
+
+namespace CharlieBackend.Api.Validators.AccountDTOValidators
+{
+    public class AuthenticationDtoValidator : AbstractValidator<AuthenticationDto>
+    {
+        public AuthenticationDtoValidator()
+        {
+            RuleFor(x => x.Email)
+                .NotEmpty().WithMessage("Email cannot be empty")
+                .EmailAddress().WithMessage("Incorrect email")
```

Comment:
Perhaps we could create string constants for common string messages like these. Because they are used almost in all of these DTO validators.

Reply:
that's what I meant by refactoring

Reply:
Alrighty then.